pyfr/backends/openmp/compiler.py

```diff
@@ -85,6 +89,19 @@ def cc_cmd(self, srcname, libname):
         # Append any user-provided arguments and return
         return cmd + self.cflags
 
+    def cc_option(self, opts, helps=[]):
+        helps = [f'--help={h}' for h in helps] + ['--help']
```

Comment:
Can't we just run `--help` for me this gives everything including if march is supported?

Reply:
No, on the some gcc versions you have to pass an additional are to help flag to get the target info. 
This is the case on `gcc (GCC) 11.4.1 20231218` for example.